roles/common/templates/etc/serverspec/os_spec.rb

```diff
@@ -0,0 +1,149 @@
+require 'spec_helper'
+
+describe file('/etc/pam.d/login') do
+    it {should contain '^@include common-auth' }
+    it {should contain '^@include common-account' }
+    it {should contain '^@include common-session' }
+    it {should contain '^@include common-password' }
+end
+
+describe file('/etc/pam.d/common-password') do
+  it { should contain '^password [success=7 default=ignore] pam_unix.so obscure sha512' }
+end
+
+describe file('/etc/login.defs') do
+  it { should contain 'PASS_MAX_DAYS   90' }
+  it { should contain 'PASS_MIN_DAYS   1' }
```

Comment:
Should use ^ to prevent being commented out
or you could do this:
```
users = `cut -d: -f1 /etc/passwd`.split("\n")
users.each do |login|
  describe user(login) do
    its(:minimum_days_between_password_change) { should eq 1 }
    its(:maximum_days_between_password_change) { should eq 90 }
  end
end
```